Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "flurry"
 version = "0.3.1"
 authors = ["Jon Gjengset <jon@thesquareplanet.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
This would bump the minimum supported rust version of the crate, which I don't think is warranted quite yet. I'd like to leave this as-is for now.